features/bounding-box-zoeken.feature

```diff
@@ -0,0 +1,131 @@
+        Als met "GET" "/panden?bbox=<bbox>" wordt gezocht
+        Dan bevat het resultaat <resultaat> het pand met identificatie "0826100000036343"
+
+        Voorbeelden:
+        | bbox                                            | resultaat | scenario                                                               |
+        | 118300.000,404800.000,118315.606,404844.967     | wel       | coördinaat is een geometriecoördinaat                                  |
+        | 118300.000,404800.000,118315.605,404844.967     | niet      | coördinaat ligt 1mm van een geometriecoördinaat                        |
+        | 118324.681,404846.437,118400.000,404800.000     | wel       | coördinaat ligt op de verbindingslijn tussen twee geometriecoördinaten |
+        | 118324.682,404846.437,118400.000,404800.000     | niet      | ligt 0,9mm van de verbindingslijn tussen twee geometriecoördinaten     |
+        | 118325.1795,404900.0000,118400.0000,404800.0000 | wel       | verbindingslijn loopt 0,5mm naast geometriecoördinaat                  |
+        | 118325.1796,404900.0000,118400.0000,404800.0000 | niet      | verbindingslijn loopt 0,6mm naast geometriecoördinaat                  |
+        | 118300.00,404800.00,118315.60,404844.96         | niet      | in centimeters valt buiten geometrie                                   |
+        | 118325.17,404900.00,118400.00,404800.00         | wel       | in centimeters valt binnen geometrie                                   |
+        | 118300,404800,118315,404844                     | niet      | in meters valt buiten geometrie                                        |
+        | 118325,404900,118400,404800                     | wel       | in meters valt binnen geometrie                                        |
+
+    Abstract Scenario: bbox <scenario> van gezochte verblijfsobject
+        Gegeven het adresseerbaar object met identificatie "0226010000038820" heeft de volgende geometrie punt coördinaten
+        | geometriecoördinaten    |
+        | [196733.427,439931.991] |
+        Als met "GET" "/adresseerbareobjecten?bbox=<bbox>" wordt gezocht
+        Dan bevat het resultaat <resultaat> het adresseerbaar object met identificatie "0226010000038820"
+
+        Voorbeelden:
+        | bbox                                            | resultaat | scenario                                            |
+        | 196000.000,440000.000,196733.427,439931.991     | wel       | coördinaat is geometriecoördinaat                   |
+        | 196000.000,440000.000,196733.427,439931.992     | niet      | coördinaat ligt 1mm van geometriecoördinaat         |
+        | 196000.000,440000.000,196800.000,439931.992     | niet      | verbindingslijn loopt 1mm van geometriecoördinaat   |
+        | 196000.0000,440000.0000,196800.0000,439931.9914 | niet      | verbindingslijn loopt 0,4mm van geometriecoördinaat |
```

Comment:
In dit geval wordt de resource dan wel teruggegeven omdat de afstand tussen de lijn en het punt < 0,5 mm is.